### ISSUE TITLE
Import and attribution of external software

### ISSUE THREAD

**Issue by jeffjennings** [nicokurtovic/SIMIO#14]
For the JOSS review, in general it would be best to import routines from other packages that SIMIO includes in its current codebase, including `RADMC3D`, `galario`, and (I think) `frank`.  

For example [this function](https://github.com/nicokurtovic/SIMIO/blob/ba35337e5f3d6779e98fc21ba3d506d1c577fc39/codes/simio_support.py#L78) from galario - galario could be made a dependency of SIMIO, and the routine `py_sampleImage` could be imported rather than copy/pasted. Alternatively, though less preferably, the routine could be kept in SIMIO, but to adhere to the galario [LICENSE](https://github.com/mtazzari/galario/blob/master/LICENSE), this would require (see the 'Conditions' at the top-right of that page) changing the SIMIO license to the same as galario (GNU Lesser General Public License v3.0), documenting the source of the code (as you do) and stating explicitly any changes made to the version in SIMIO relative to that in galario.

Similarly, [this function](https://github.com/nicokurtovic/SIMIO/blob/ba35337e5f3d6779e98fc21ba3d506d1c577fc39/codes/simio_support.py#L26) appears to me to be copy/pasted from a [function in frank](https://github.com/discsim/frank/blob/5ad9382470c5ecadb9f29b5bd6db0995943ff6aa/frank/geometry.py#L82), with slight modification (but I don't see any attribution to frank). Again to keep this in the SIMIO code base would require changing the SIMIO license, documenting the source of the code, and explicitly noting the changes made. Altenatively you couldmake frank a dependency, import the routine from it, and modify the output as needed.

[https://github.com/openjournals/joss-reviews/issues/4942](https://github.com/openjournals/joss-reviews/issues/4942)

**Comment by nicokurtovic** [comment]
I am very sorry I did not mention that the projection and deprojection of the visibilities was initially based on a frank function. I modified it some time ago to include the shifting in space, and it was my mistake to forget where it originally came from. I should have mentioned it, as I also mentioned the origin of other functions. 

I changed the structure of the deprojection function. The shifting, rotation, projection, and deprojection is done in separate tasks, which are all called from the [handle_uv](https://github.com/nicokurtovic/SIMIO/blob/main/codes/simio_support.py#L117) function. 

I decided to exclude galario and the functions calling galario from SIMIO. Using galario was only optional in the package, and now the visibilities are only calculated with the Fourier Transform of CASA.
